shared/devices/row/container.js

```diff
@@ -26,16 +27,12 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => ({
   firstItem: ownProps.firstItem,
   isCurrentDevice: stateProps.isCurrentDevice,
   isRevoked: stateProps.isRevoked,
+  isNew: stateProps.isNew,
   name: stateProps.name,
   showExistingDevicePage: () => dispatchProps._showExistingDevicePage(ownProps.deviceID),
   type: stateProps.type,
 })
 
-export default compose(
-  connect(
-    mapStateToProps,
-    mapDispatchToProps,
-    mergeProps
-  ),
-  setDisplayName('DeviceRow')
-)(DeviceRow)
+export default compose(connect(mapStateToProps, mapDispatchToProps, mergeProps), setDisplayName('DeviceRow'))(
```

Comment:
you likely don't have prettier installed?

Reply:
lol. i thought it was prettier that added this to my diff.

Reply:
if you're using vscode the team has been complaining that its not doing the right thing re: prettier since the last update

Reply:
i am. i can change this back to what it was. /shrug